doc/source/user_guide/groupby.rst

```diff
@@ -433,7 +437,7 @@ Iterating through groups
 ------------------------
 
 With the GroupBy object in hand, iterating through the grouped data is very
-natural and functions similarly to :py:func:`itertools.groupby`:
+natural and works similarly to :py:func:`itertools.groupby`:
```

Comment:
What is the benefit of changing this?

Reply:
IMO "works" sounded more natural. 
I can leave it as it was.